jobtracker.go

```diff
@@ -422,7 +424,7 @@ func (jt *jobTracker) FetchJobDetails(id string, host string) (*jobDetail, error
 	var url string
 	appID, jobID := hadoopIDs(id)
 	if host == jt.rm {
-		url = fmt.Sprintf("%s/proxy/%s/ws/v1/mapreduce/jobs", host, appID)
+		url = fmt.Sprintf("%s/proxy/%s/ws/v1/mapreduce/jobs", jt.ps, appID)
```

Comment:
This looks confusing given the previous line. In a proxyserver world, is there any reason to know about the separation between the proxy and the resourcemanager? Is there anything that doesn't work if you pass the proxyserver URL as the resourcemanager URL?